admiral/envs/wrappers/ravel_discrete_wrapper.py

```diff
@@ -113,6 +113,7 @@ def _isbounded(space):
         not (space.high == np.iinfo(space.dtype).min).any() and \
         not (space.high == np.iinfo(space.dtype).max).any()
 
+# TODO: should I add bounded feature to gym util and use that instead?
```

Comment:
Resolve this todo

Reply:
`check_space` in `ravel_discrete_wrapper` has specific rules for bounded integer Box space that are needed for checking that the space can be ravelled but do not apply to spaces in general, so we will leave the function here as is.